docs/spring-boot/core-starter.md

```diff
@@ -213,6 +213,14 @@ public class CoreServices {
 }
 ```
 
+Unlike subscribing to real time events, using slash commands or activities, solely injecting services does not require
+the datafeed loop to run. If you want to disable the datafeed loop, you can update your `application.yaml` file as follows:
```

Comment:
so this is maybe how we should call the DataFeedService class @symphony-hong (if it is referred to as the datafeed loop)

Reply:
maybe we need to be more explicit about what won't work if you use it? (for instance slash commands?)

Reply:
> Unlike subscribing to real time events, using slash commands or activities
This is what won't work if you disable the datafeed loop. Maybe I can come up with a more explicit sentence.

Reply:
pwahhhh my bad I commented too quickly